include/fast_float/decimal_to_binary.h

```diff
@@ -189,6 +189,108 @@ adjusted_mantissa compute_float(int64_t q, uint64_t w)  noexcept  {
   return answer;
 }
 
+// w * 2 ** q
+template <typename binary>
+fastfloat_really_inline
+adjusted_mantissa compute_hex_float(int64_t q, uint64_t w)  noexcept {
+  adjusted_mantissa answer;
+  if (w == 0) {
```

Comment:
You probable want to add a fast path if the mantissa can be represented correctly that uses `ldexp`